DOC-1325: RS CLI Reorg: rladmin node (#1920)
RedisLabs/redislabs-docs: content/rs/references/cli-utilities/rladmin/node/addr.md

---
Title: rladmin node addr set
linkTitle: addr
description: Sets the internal IP address of a node.
weight: $weight
alwaysopen: false
toc: "true"
headerRange: "[1-2]"
categories: ["RS"]
aliases:
---

> `rladmin node addr set` sets the internal IP address of a node. The internal IP address can only be set with `rladmin node addr` when the node is down.

Reviewer: ```suggestion
`rladmin node addr set` sets the internal IP address of a node. You can set the internal IP address using `rladmin node addr` only when the node is down.
```

```sh
rladmin node <id>
        addr set <ip_addr>
```

### Parameters

| Parameter | Type/Value                     | Description                                                                                   |
|-----------|--------------------------------|-----------------------------------------------------------------------------------------------|

> | node      | integer                        | Sets internal IP address of the specified node                                                |
> | ip_addr   | IP address                     | Sets specified IP address as the internal IP of the node                                      |

Reviewer 2: ```suggestion
| node      | integer                        | Sets the internal IP address of the specified node                                                |
| ip_addr   | IP address                     | Sets the node's internal IP address to the specified IP address                                      |
```

### Returns

Returns `Updated successfully` if the IP address was set.

> Use [`rladmin status nodes`]({{<relref "/rs/references/cli-utilities/rladmin/status#status-nodes">}}) to verify the internal IP address was changed.

Reviewer: Consider starting with action first, i.e. "Verify that the internal IP address has changed, ..."
